src/index.ts

```diff
@@ -893,9 +906,19 @@ class OpenSIPSJS extends UA {
         call.roomId = roomId
         call.localMuted = false
 
-        this._addCall(call)
+        if (call.direction === 'incoming' && this.autoAnswer) {
+            this._addCall(call, false)
+        } else {
+            this._addCall(call)
+        }
+
+        // this._addCall(call)
         this._addCallStatus(session.id)
         this._addRoom(newRoomInfo)
+
+        if (call.direction === 'incoming' && this.autoAnswer) {
```

Comment:
And here you can reuse variable from comment above:

```suggestion
        if (doAutoAnswer) {
```